layout/settings/settings_gameplay.xml

```diff
@@ -195,47 +212,59 @@
 
 				<!-- TODO: infomessages here in the future. Not doing now because of future safeguard changes -->
 
-				<ChaosSettingsEnumDropDown text="#Settings_Safeguards_PracticeMode" convar="mom_run_safeguard_practicemode">
+				<ChaosSettingsEnumDropDown
+					text="#Settings_Safeguards_PracticeMode"
+					convar="mom_run_safeguard_practicemode"
+					tags="practice"
```

Comment:
Should these safeguards settings each have a tag for safeguards? Or is it sufficient to have the header use that tag only?

Reply:
Yep, that's exactly the idea! The header tags are parsed out of the group title labels, that then applies to everything within the group